src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/git.rb

```diff
@@ -13,7 +14,7 @@ def self.last_tag
   end
 
   def self.last_release_tag
-    command = 'git tag | sort -r | grep -v unstable | head -1'
+    command = "git tag | sort -r | egrep '#{RELEASE_REGEX}' | head -1"
```

Comment:
Shouldn't this receive a param instead to match on the prerelease_identifier that was passed to calculate_next_canary_version.rb?

Reply:
We might have different pre-release identifiers (likely `preview` before GA, and `dev`or `unstable` after), so I wanted to filter out all pre-releases in general. I think in practice `preview` versions will likely all have very low version numbers, so only filtering with one of the identifiers will still work, though not semantically correct.

Reply:
Got it, that's fair

Reply:
I just realized that the current `last_release_tag` using `sort` doesn't seem to correctly handle the ordering of versions with more than one digit, for example `v0.10.0` and `v0.9.0`.

To correctly order the tags, the best way seems to be parsing them into `Version`s and use the ordering defined for `Version`s. However, the current code expects `Git.last_release_tag` to return a string, and converts it into a `Version` in `calculate_next_release_version`. It is not used anywhere else.

Would it be ok if we change `Git.last_release_tag` to return a `Version` instead of a string (and rename it to `last_release_version`), so we can avoid converting back and forth?